local/blockchain.go

```diff
@@ -423,41 +423,41 @@ func (ln *localNetwork) getCurrentSubnets(ctx context.Context) ([]ids.ID, error)
 }
 
 // TODO: make this "restart" pattern more generic, so it can be used for "Restart" RPC
-func (ln *localNetwork) restartNodesWithWhitelistedSubnets(
+func (ln *localNetwork) restartNodesWithTrackSubnets(
 	ctx context.Context,
 	subnetIDs []ids.ID,
 ) (err error) {
 	fmt.Println()
-	ln.log.Info(logging.Green.Wrap("restarting each node"), zap.String("whitelisted-subnets", config.WhitelistedSubnetsKey))
-	whitelistedSubnetIDsMap := map[string]struct{}{}
+
```

Comment:
I suggest to keep this log but in Debug

Reply:
this log was kinda useless. it is printing the name of config.WhitelistedSubnetsKey flag

Reply:
It was also printing "restarting each node" which seems useful in Debug. Nevertheless, leave it up to you.

Reply:
We have a very similar log some lines afterwards, which also prints the tracked ids